src/components/playbackSettings/playbackSettings.template.html

```diff
@@ -157,6 +157,17 @@ <h2 class="sectionTitle">
         <div class="selectContainer">
             <select is="emby-select" class="selectSkipBackLength" label="${LabelSkipBackLength}"></select>
         </div>
+        <div class="selectContainer">
+            <select is="emby-select" id="selectPreferredTranscodeAudioCodec" label="${LabelSelectPreferredTranscodeAudioCodec}">
+                <option value="">${Auto}</option>
+                <option value="aac">AAC</option>
+                <option value="ac3">AC3</option>
+                <option value="dts">DTS</option>
+                <option value="eac3">EAC3</option>
+                <option value="flac">FLAC</option>
+                <option value="opus">Opus</option>
```

Comment:
ALAC is also a supported codec in 10.9:
```suggestion
                <option value="opus">Opus</option>
                <option value="alac">ALAC</option>
```